### PR TITLE
Ephemeral rules script

### PR DESCRIPTION
**Pull request by JulianaCP-v2**

### CODE REVIEW

File: operator_toolkit/create_soc_percentile.py [hunk 1]
```diff
@@ -65,8 +65,8 @@ def socs_field_names(self):
                 self.fieldnames.append(f'{site_id}_{sg[0]}')
         return self.fieldnames
 
-    def create_schedule(self, sites_to_operate):
-        time_step = datetime.datetime(2021, 5, 21, 14, 55)
+    def create_schedule(self, sites_to_operate, schedule_datetime):
```
Comment (lucasmlofaro) [inline operator_toolkit/create_soc_percentile.py:68]: Is there a better name than `schedule_datetime`? Like is this the start of the schedule or is there some other descriptive term we could use?

Comment (lucasmlofaro) [reply operator_toolkit/create_soc_percentile.py:68]: Aha, it's in the help text
```
End time of the operation
```
Maybe `end_time` is good enough, but open to suggestions.

File: operator_toolkit/ephemeral_bound_schedule.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+from common import (
+    CLI_MESSAGE,
+    sites,
+    AXIOM_API_AUTH_USER,
+    AXIOM_API_AUTH_PASSWORD,
+    load_file,
+    basic_url,
+    ServiceType,
+)
```
Comment (lucasmlofaro) [inline operator_toolkit/ephemeral_bound_schedule.py:17]:
```suggestion
from common import (
    AXIOM_API_AUTH_USER,
    AXIOM_API_AUTH_PASSWORD,
    CLI_MESSAGE,
    ServiceType,
    sites,
    load_file,
    basic_url,
)
```

File: operator_toolkit/ephemeral_bound_schedule.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+
+def create_ephemeral_schedule(site, start, stop, bound_multiplier, soc_25_percentile):
```
Comment (lucasmlofaro) [inline operator_toolkit/ephemeral_bound_schedule.py:20]: Perhaps just a note for later, but this feels to me like it should be a method in a class. I'm wondering if we should schedule some time to go through all the scripts and decide on a better data model for some of these things.

File: operator_toolkit/create_soc_percentile.py (hunk 1 above)
Comment (lucasmlofaro) [reply operator_toolkit/create_soc_percentile.py:68]: Looks like we just use `end` in the other script from this PR.

**Review by lucasmlofaro — CHANGES_REQUESTED** [review]
Looks pretty good, a couple of minor things.

File: operator_toolkit/create_soc_percentile.py (hunk 1 above)
Comment (JulianaCP-v2) [reply operator_toolkit/create_soc_percentile.py:68]: good catch! I copy and paste the help description from the previous `end` args param and I forgot to update this one. in theory, this `schedule_datetime` is a different value from `start` or `end` datetimes. This specifically is used to create a `schedule.csv` file.

File: operator_toolkit/ephemeral_bound_schedule.py (hunk 3 above)
Comment (JulianaCP-v2) [reply operator_toolkit/ephemeral_bound_schedule.py:20]: yes, definitely. I'm feeling the same as you. In this next sprint, I'm thinking about creating some classes to process these scripts so we can improve in general the process (re use code, etc)

File: operator_toolkit/create_soc_percentile.py (hunk 1 above)
Comment (JulianaCP-v2) [reply operator_toolkit/create_soc_percentile.py:68]: Actually I just spoke with Eliav, he told me he doesn't need that schedule csv file any more, So I'm just going to remove all related code with that

**Review by lucasmlofaro — APPROVED** [review]

**Pull request merged**